app/policies/recipe_policy.rb

```diff
@@ -0,0 +1,53 @@
+class RecipePolicy < ApplicationPolicy
+  attr_reader :user, :recipe
+
+  def initialize(user, recipe)
+    @user = user
+    @recipe = recipe
+  end
+
+  def show
+    owner?
+  end
+
+  def index
```

Comment:
Wrong name here as well. I'm also skeptical of the `owner?` check for this action since you will have a collection of recipes instead of just one. Technically, you aren't even calling authorize on the index.  I'd probably just make this return `true`.
```suggestion
  def index?
```